app/assets/javascripts/discourse-markdown-it/src/index.js

```diff
@@ -51,6 +51,12 @@ export default class DiscourseMarkdownIt {
   }
 
   cook(raw) {
+    if (raw === undefined || raw === null) {
+      return "";
+    }
+
+    raw = raw.toString();
+
```

Comment:
This change allows data that can be interpreted as strings (Eg, `SafeString`) to be cooked. This is needed for the `<CookText />` element in `reviewable-queued-post.gjs` to work properly.